supportedBackends/sfs/src/main/scala/cromwell/backend/sfs/SharedFileSystem.scala

```diff
@@ -23,7 +24,7 @@ object SharedFileSystem extends StrictLogging {
   import better.files._
 
   final case class AttemptedLookupResult(name: String, value: Try[WdlValue]) {
-    def toPair = name -> value
+    def toPair: (String, Try[WdlValue]) = name -> value
```

Comment:
Well nowadays it keeps IntelliJ from yellowing by default, though arguably we should turn off that default behavior if we don't like it.